<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>configuring_network_guiClass</class>
 <widget class="QDialog" name="configuring_network_guiClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>20</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>DFSense网络配置</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <property name="horizontalSpacing">
        <number>20</number>
       </property>
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButton_find">
         <property name="text">
          <string>搜索相机</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="pushButton_auto">
         <property name="text">
          <string>自动配置</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableWidget" name="tableWidget_network_message">
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="midLineWidth">
        <number>1</number>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderDefaultSectionSize">
        <number>43</number>
       </attribute>
       <attribute name="verticalHeaderMinimumSectionSize">
        <number>43</number>
       </attribute>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="configuring_network_gui.qrc"/>
 </resources>
 <connections/>
</ui>
